styles/pinterest/catppuccin.user.css

```diff
@@ -193,12 +195,24 @@
       color: var(--color-text-inverse);
     }
 
+    // Profile button
+    // Invite collaborators buttons
+    div[style="border: 1px solid white;"],
     div[data-test-id="gestalt-avatar-svg"] {
-      box-shadow: 0 0 0 1px @crust !important;
+      box-shadow: 0 0 0 0 @surface1 !important;
+      outline-color: @base !important;
+      outline-width: 0 !important;
```

Comment:
do we need this line?

Reply:
I don't see any change from box-shadow, we can remove that.